precompiles/assets-erc20/src/lib.rs

```diff
@@ -325,8 +321,8 @@ where
 		let mut input = handle.read_input()?;
 		input.expect_arguments(2)?;
 
-		let spender: H160 = input.read::<Address>()?.into();
-		let amount: U256 = input.read()?;
+		let spender: H160 = input.read::<Address>().in_field("spender")?.into();
+		let amount: U256 = input.read().in_field("value")?;
```

Comment:
```suggestion
		let amount: U256 = input.read().in_field("amount")?;
```
so there is no compile error if the `in_field` argument doesn't match the variable name

Is there a way to use the variable name by default as argument or is that too complicated?